<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdminPage</class>
 <widget class="QDialog" name="AdminPage">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>336</width>
    <height>432</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>301</width>
     <height>141</height>
    </rect>
   </property>
   <property name="text">
    <string>Вы были назначены администратором, введите параметры игры: высоту и ширину поля, количество игроков, таймаут, количество ходов и имя комнаты</string>
   </property>
   <property name="scaledContents">
    <bool>false</bool>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="pushButton">
   <property name="geometry">
    <rect>
     <x>240</x>
     <y>390</y>
     <width>88</width>
     <height>34</height>
    </rect>
   </property>
   <property name="text">
    <string>Ок</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>120</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="lineEdit_1">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>160</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cmd">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>200</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cmd_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>240</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cmd_3">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>280</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="Cmd_4">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>320</y>
     <width>311</width>
     <height>32</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
